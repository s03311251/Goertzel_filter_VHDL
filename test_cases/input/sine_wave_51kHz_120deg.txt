3BB6
3542
2CA5
22BE
1890
0F23
076E
0239
000C
0120
0559
0C48
1539
1F43
2960
3289
39CE
3E73
3FFF
3E48
397C
3218
28DC
1EBA
14B8
0BDC
050E
00FD
0015
026C
07C7
0F98
1915
2347
2D23
35A8
3BF9
3F71
3FB6
3CBF
36DC
2EA5
24F0
1ABA
110E
08E7
031A
003E
009E
0430
0A96
132B
1D0E
273D
30B0
3870
3DB3
3FF0
3EEC
3AC3
33E0
2AF7
20F0
16D0
0DA0
064F
019C
0000
01A6
0664
0DBC
16F1
2112
2B17
33FA
3AD5
3EF5
3FEE
3DA6
385A
3093
271C
1CEC
130B
0A7D
041F
0098
0043
0329
08FF
112C
1ADC
2512
2EC3
36F4
3CCE
3FBA
3F6B
3BE8
358F
