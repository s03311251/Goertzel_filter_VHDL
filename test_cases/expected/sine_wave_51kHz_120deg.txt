000EA920
0009C3A0
